--- brltty-4.5/configure.ac.orig	2013-03-27 23:54:58.000000000 +0100
+++ brltty-4.5/configure.ac	2013-04-07 19:04:31.636980920 +0200
@@ -1516,7 +1516,7 @@
    BRLTTY_SPEECH_DRIVER([fv], [Festival])
    BRLTTY_SPEECH_DRIVER([gs], [GenericSay])
    BRLTTY_IF_PACKAGE([Mikropuhe], [mikropuhe], [mpwrfile.h], [BRLTTY_IF_PTHREADS([BRLTTY_SPEECH_DRIVER([mp], [Mikropuhe], [-L$(MPLINUX_ROOT) -lmplinux])])])
-   BRLTTY_IF_PACKAGE([speech-dispatcher], [speechd], [include/libspeechd.h], [BRLTTY_SPEECH_DRIVER([sd], [SpeechDispatcher], [-L$(SPEECHD_ROOT)/lib -lspeechd])])
+   BRLTTY_IF_PACKAGE([speech-dispatcher], [speechd], [include/speech-dispatcher/libspeechd.h], [BRLTTY_SPEECH_DRIVER([sd], [SpeechDispatcher], [-lspeechd])])
    BRLTTY_IF_PACKAGE([Swift], [swift], [include/swift.h], [BRLTTY_SPEECH_DRIVER([sw], [Swift], [-L$(SWIFT_ROOT)/lib -lswift -lm])])
    BRLTTY_IF_PACKAGE([Theta], [theta], [include/theta.h], [BRLTTY_SPEECH_DRIVER([th], [Theta], [-L$(THETA_ROOT)/lib -ltheta])])
    BRLTTY_IF_PACKAGE([ViaVoice], [viavoice], [include/eci.h], [BRLTTY_SPEECH_DRIVER([vv], [ViaVoice], [-L$(VIAVOICE_ROOT)/lib -libmeci50])])
#--- brltty-4.5/Drivers/Speech/SpeechDispatcher/Makefile.in.orig	2013-03-27 23:51:36.000000000 +0100
#+++ brltty-4.5/Drivers/Speech/SpeechDispatcher/Makefile.in	2013-04-07 19:41:55.110267490 +0200
#@@ -25,5 +25,5 @@
# include $(SRC_TOP)speech.mk
# 
# speech.$O:
#-	$(CC) $(SPK_CFLAGS) -I$(SPEECHD_ROOT)/include -c $(SRC_DIR)/speech.c
#+	$(CC) $(SPK_CFLAGS) -I$$(pkg-config --cflags speech-dispatcher) -c $(SRC_DIR)/speech.c
# 
--- brltty-4.5/Drivers/Speech/SpeechDispatcher/speech.c.orig	2013-03-27 23:51:36.000000000 +0100
+++ brltty-4.5/Drivers/Speech/SpeechDispatcher/speech.c	2013-04-07 19:56:47.340248894 +0200
@@ -38,7 +38,7 @@
 #define SPK_HAVE_PUNCTUATION
 #include "spk_driver.h"
 
-#include <libspeechd.h>
+#include <speech-dispatcher/libspeechd.h>
 
 static SPDConnection *connectionHandle = NULL;
 static const char *moduleName;
