diff -ur brltty-3.10.orig/aclocal.m4 brltty-3.10/aclocal.m4
--- brltty-3.10.orig/aclocal.m4	2008-07-17 00:19:54.000000000 +0200
+++ brltty-3.10/aclocal.m4	2008-10-07 09:38:51.000000000 +0200
@@ -143,8 +143,8 @@
 fi])
 
 AC_DEFUN([BRLTTY_HELP_STRING], [dnl
-AC_HELP_STRING([$1], patsubst([$2], [
-.*$]), [brltty_help_prefix])dnl
+AS_HELP_STRING([$1], patsubst([$2], [
+.*$]), brltty_help_indent)dnl
 patsubst(patsubst([$2], [\`[^
 ]*]), [
 ], [\&brltty_help_prefix])[]dnl
diff -ur brltty-3.10.orig/configure.ac brltty-3.10/configure.ac
--- brltty-3.10.orig/configure.ac	2008-07-17 00:19:54.000000000 +0200
+++ brltty-3.10/configure.ac	2008-10-07 09:46:25.000000000 +0200
@@ -16,7 +16,7 @@
 # This software is maintained by Dave Mielke <dave@mielke.cc>.
 ###############################################################################
 
-AC_PREREQ([2.53])
+AC_PREREQ([2.62])
 AC_INIT([brltty], [3.10], [http://mielke.cc/brltty/])
 AC_CONFIG_SRCDIR([Programs/brltty.h])
 BRLTTY_SUMMARY_BEGIN
@@ -779,7 +779,7 @@
 case "${host_os}"
 in
    linux*|gnu*|kfreebsd*)
-      brltty_cv_prog_cc_sysflags="-D_POSIX_C_SOURCE=2 -D_BSD_SOURCE -D_XOPEN_SOURCE=500 -D_XOPEN_SOURCE_EXTENDED"
+      brltty_cv_prog_cc_sysflags="-D_POSIX_C_SOURCE=2 -D_BSD_SOURCE -D_XOPEN_SOURCE=500 -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE"
       ;;
    solaris*)
       brltty_cv_prog_cc_sysflags="-D_XOPEN_SOURCE=500 -D__EXTENSIONS__"
