diff -urNp -x '*.orig' brltty-6.1.org/Programs/brlapi_client.c brltty-6.1/Programs/brlapi_client.c
--- brltty-6.1.org/Programs/brlapi_client.c	2020-04-05 12:35:34.000000000 +0200
+++ brltty-6.1/Programs/brlapi_client.c	2021-03-02 22:36:23.483472412 +0100
@@ -28,6 +28,7 @@
 #include <errno.h>
 #include <fcntl.h>
 #include <sys/stat.h>
+#include <sys/sysmacros.h>
 #include <locale.h>
 #include <assert.h>
 #include <limits.h>
